src/main/java/uk/gov/ons/census/action/model/entity/ActionType.java

```diff
@@ -15,8 +15,10 @@
   CE1_IC01(ActionHandler.PRINTER, "D_CE1A_ICLCR1"),
   CE1_IC02(ActionHandler.PRINTER, "D_CE1A_ICLCR2B"),
 
-  // Individual addressed initial contact letters for CE Units
+  // Individual addressed initial contact letters for CE Estabs and Units
+  CE_IC03(ActionHandler.PRINTER, "D_ICA_ICLR1"),
   CE_IC03_1(ActionHandler.PRINTER, "D_ICA_ICLR1"),
+  CE_IC04(ActionHandler.PRINTER, "D_ICA_ICLR2B"),
   CE_IC04_1(ActionHandler.PRINTER, "D_ICA_ICLR2B"),
```

Comment:
Can the estab and unit types be split up with comment for each so that we know which is which?